Bottom: 23d1b8ac262ad8c25aa048ce092d0a3bf43a14cf
Top:    e999ca420e4e25de8ead8db3280a3c319101906a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2017-02-26 14:14:07 +0000

Refresh of speedup-for-models-whilst

---

diff --git a/src/polygon.h b/src/polygon.h
index 664d885..c50079f 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -35,7 +35,7 @@
 /* Implementation constants */
 
 #define POLY_CIRC_SEGS 8
-#define POLY_CIRC_SEGS_D (double)POLY_CIRC_SEGS)
+#define POLY_CIRC_SEGS_D ((double)POLY_CIRC_SEGS)
 
 #if 0
 /* THIS IS BROKEN:
